src/main/scala/io/iohk/ethereum/network/handshaker/EtcNodeStatusExchangeState.scala

```diff
@@ -21,49 +18,16 @@ case class EtcNodeStatusExchangeState(handshakerConfiguration: EtcHandshakerConf
       timeout = peerConfiguration.waitForStatusTimeout
     )
 
-  def applyResponseMessage: PartialFunction[Message, HandshakerState[PeerInfo]] = { case remoteStatus: Status =>
```

Comment:
There seems to be quite repeated code between `EtcNodeStatus64ExchangeState` and `EtcNodeStatus64ExchangeState` could we maybe add a function here `applyRemoteStatusMessage: RemoteStatus => HandshakerState[PeerInfo]`? The states would then only convert Status to RemoteStatus and call it